airgun/entities/contentview_new.py

```diff
@@ -0,0 +1,48 @@
+from navmazing import NavigateToSibling
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep
+from airgun.navigation import navigator
+from airgun.utils import retry_navigation
+from airgun.views.contentview_new import NewContentViewCreateView
+from airgun.views.contentview_new import NewContentViewTableView
+
+
+class NewContentViewEntity(BaseEntity):
+    endpoint_path = '/content_views'
+
+    def create(self, values, composite=False):
+        """Create a new content view"""
+        view = self.navigate_to(self, 'New')
+        if composite:
```

Comment:
Why is this filled separately from the rest of `values`?   It should fill naturally as part of filling the view.

If it needs to be set before any other values are set, I would use `before_fill` for that, because it's related to the view behavior, not the entity creation behavior.

Reply:
@mshriver This is fixed with your suggestion a few weeks back.  I'm having issues testing this with the correct syntax.  I'm using this and it's not correct: `session.contentview_new.create({'name': 'sample', 'composite_view': {True}})`.  What's the correct syntax?